src/decoder_allocators.cpp

```diff
@@ -38,7 +38,7 @@ zmq::shared_message_memory_allocator::shared_message_memory_allocator(size_t buf
         bufsize( 0 ),
         max_size( bufsize_ ),
         msg_refcnt( NULL ),
-        maxCounters( std::ceil( (double)max_size / (double)msg_t::max_vsm_size) )
+        maxCounters( std::ceil( static_cast<double>(max_size) / static_cast<double>(msg_t::max_vsm_size)) )
```

Comment:
I've seen other code spell this like `static_cast <double` -- Pieter, what's the style rule on the cast syntax?